src/portals/hope/pages/paymentOrders/components/SelectedOrderSidePanel.tsx

```diff
@@ -207,37 +260,73 @@ export const SelectedOrderSidePanel: React.FC<{
               </div>
               <div>
                 <Label>Deductible</Label>
-                <Monetary amount={order.deductible} />
+                <Popover contents="Copy deductible">
+                  <Info
+                    onClick={() => {
+                      copy(`${parseFloat(order.deductible.amount)}`)
+                      toast.success('Deductible copied')
+                    }}
+                  >
+                    <Monetary amount={order.deductible} />
+                  </Info>
+                </Popover>
               </div>
             </Row>
           </>
         </EditSection>
       )}
-
       <EditSection>
         <strong>Notes</strong>
         {order.notes.map((note) => (
-          <div key={note.addedAt}>{note.text}</div>
+          <Flex justify="space-between" gap="small" key={note.addedAt}>
+            <span>{note.text}</span>
+            <div style={{ textAlign: 'right' }}>
+              <div>{format(parseISO(note.addedAt), 'yyyy-MM-dd HH:mm:ss')}</div>
+              {note.addedByUser.__typename === 'AdminSystemUser' ? (
+                <div>{note.addedByUser.email}</div>
+              ) : (
+                <div>Automation</div>
+              )}
+            </div>
+          </Flex>
         ))}
       </EditSection>
-
       {!(order.state === 'SETTLED' || order.state === 'DISMISSED') && (
         <EditSection>
           <Row columns={2}>
             {order.state === 'APPROVED' ? (
               <>
                 <Button
-                  onClick={() =>
-                    confirm('Do you want to settle?').then(() =>
-                      settlePaymentOrder(order.id),
-                    )
-                  }
+                  onClick={(e) => {
+                    e.stopPropagation()
+                    confirmWithValue({
+                      content: 'Settle payment order',
+                      type: 'checkbox',
+                      label: 'Also close claim',
+                    }).then((closeClaim) => {
+                      settlePaymentOrder(order.id).then(() => {
```

Comment:
To make this generic
```suggestion
                      settlePaymentOrder(order.id, order.claimId).then(() => {
```